app/controllers/spree/admin/orders/customer_details_controller_decorator.rb

```diff
@@ -1,6 +1,22 @@
 Spree::Admin::Orders::CustomerDetailsController.class_eval do
   before_filter :set_guest_checkout_status, only: :update
 
+  def update
+    if @order.update_attributes(params[:order])
+      if params[:guest_checkout] == "false"
```

Comment:
I'm a bit surprised to compare a boolean value to a string. Rails does parse JSON to Ruby primitives. How is it possible that we end up with a String and not with a FalseClass instance 🤔 ? I would have expected to read this as `if !params[:guest_checkout]`

Reply:
@sauloperez This isn't the case when the string is sent to the server as HTTP form data. However, this string would still be converted to a boolean as we expect when assigned to an `ActiveRecord` boolean attribute. :slightly_smiling_face: